tomcat/content/thredds/threddsConfig.xml

```diff
@@ -202,6 +202,7 @@
   <WMS>
     <allow>true</allow>
     <allowRemote>false</allowRemote>
+    <paletteLocationDir>palettes</paletteLocationDir>

```

Comment:
Got this methodology from: https://www.unidata.ucar.edu/software/tds/v4.5/reference/ThreddsConfigXMLFile.html#WMS